api/src/api/fireSafetyRouter.ts

```diff
@@ -0,0 +1,21 @@
+import { FireSafetyInspectionStatus } from "@domain/types";
+
+import { FireSafetyInspectionResult } from "@shared/fireSafety";
+import { Router } from "express";
+
+export const fireSafetyRouterFactory = (fireSafetyInspection: FireSafetyInspectionStatus): Router => {
+  const router = Router();
+
+  router.get("/fire-safety/:address", async (req, res) => {
+    const address = req.params.address as string;
```

Comment:
[sand] I know we don't have a ui designed yet, but will the address be a single string? Will it not have like a street address 1, zipcode, city? If so we might have rethink how this is parameterized but if not then nevermind.

Reply:
Right now we have just been doing address but this is a great point, I can definitely expand this with named parameters if we expand like that